frontend/packages/dsco/src/divider/README.md

```diff
@@ -0,0 +1,16 @@
+# `componentLibrary/divider`
+
+## Consuming This Component
+
+This package exports one styled React component: [Divider](Divider.tsx).
+You can import it like this:
+
+```javascript
+import Divider from '@code-dot-org/component-library/divider';
+```
+
+For guidelines on how to use this component and the features it
+offers, [visit Storybook](https://code-dot-org.github.io/dsco_)
+(link to be updated once code-dot-org storybook will be public.).
+Or run storybook locally and go
+to [Atoms / Divider](http://localhost:9001/?path=/docs/designsystem-divider--docs).
```

Comment:
nit: right now it's [Design System / Divider] based on what I see in url and in stories code.
We can update it to be [Atoms / Divider] later if needed

Reply:
Ah good catch, I'll update this to be [Design System / Divider] since it'll be used in DSCO broadly.